components/cam/src/physics/cam/nudging.F90

```diff
@@ -151,6 +223,12 @@ module nudging
 !      Nudge_Vwin_Hindex   - REAL HI model index of transition
 !      Nudge_Vwin_Ldelta   - REAL LO transition length
 !      Nudge_Vwin_Hdelta   - REAL HI transition length
+!      Nudge_Method        - CHAR method to perform Nudging analyses. [Step,Linear,IMT]
+!      Nudge_Tau           - REAL relaxation time scale if Nudge_Tau > 0
+!      Nudge_Loc_Same      - LOGICAL change the location of calculation of nudging tendency 
+!      Nudge_Curr          - LOGICAL linearly interpolate nudging data to current
+!                                    or future model time step
+!      Nudge_File_Ntime    - INT  Number of time slices per nudging data file
 !    /
```

Comment:
There are so many options in this nudging_nl configuration. It is more efficient to demonstrate the usage by examples. Here seems the right place to include 1 or 2 examples with advanced settings (similar to Table 3 of https://www.geosci-model-dev.net/12/2679/2019/, but with modifications accordingly).

Reply:
Hi Qi, thanks for this suggestion. Please see the example settings for the revised nudging codes in your first comment.

Reply:
The Table in my paper doesn't have the new settings added by this PR. Can you make a new one and add it as comments in the code here?

Reply:
Thanks, @tangq . I have added two example namelist settings for the typical global nudged simulation and CONUS RRM simulation (based on the Table 3 of your paper) as comments in the nudging.F90. It could be viewed at the commit **f78268c**.